code/modules/bitrunning/objects/loot_crate.dm

```diff
@@ -0,0 +1,89 @@
+#define ORE_MULTIPLIER_IRON 3
+#define ORE_MULTIPLIER_GLASS 2
+#define ORE_MULTIPLIER_PLASMA 1
+#define ORE_MULTIPLIER_SILVER 0.7
+#define ORE_MULTIPLIER_GOLD 0.6
+#define ORE_MULTIPLIER_TITANIUM 0.5
+#define ORE_MULTIPLIER_URANIUM 0.4
+#define ORE_MULTIPLIER_DIAMOND 0.3
+#define ORE_MULTIPLIER_BLUESPACE_CRYSTAL 0.2
```

Comment:
I would split this up into a tier system, which is what I did in terms of mineral rarity on my mining PR.
As it stands, iron/glass are one tier, silver/titanium are another,  gold/plasma/uranium being the 3rd, with the rarest tier being BS/diamonds. As it stands we treat at lot of these minerals the same in terms of output for recipe purposes.

Reply:
this is already tiered via populate_contents